gapic/src/main/com/google/gapid/models/Devices.java

```diff
@@ -99,21 +102,59 @@ public void onCaptureLoaded(Loadable.Message error) {
   protected void resetReplayDevice() {
     replayDevices = null;
     selectedReplayDevice = null;
+    incompatibleReplayDevices = null;
+  }
+
+  public class ReplayDeviceInfo {
```

Comment:
This should be static. And can you move it to the bottom please. I'd say, put it right after the `Listener` interface.

Reply:
done